src/main.ts

```diff
@@ -75,7 +75,8 @@ app.config.globalProperties.$filters = {
       featureHierarchy.filter((featureItem: any) => featureItem.startsWith(featureKey)).forEach((feature: any) => {
         const featureSplit = feature ? feature.split('/') : [];
         const featureValue = featureSplit[2] ? featureSplit[2] : '';
-        featuresValue +=  " " + featureValue;
+        featuresValue += ", " + featureValue;
+        featuresValue = featuresValue.slice(1);
```

Comment:
I think we should find a much better approach to add a comma between values.

Reply:
Yes Sir. Updated code with a better approach.